tests/Integration/Database/EloquentPrunableTest.php

```diff
@@ -45,16 +45,14 @@ public function testPrunesRecords()
     {
         Event::fake();
 
-        collect(range(1, 5000))->map(function ($id) {
+        PrunableTestModel::insert(collect(range(1, 50))->map(function ($id) {
             return ['name' => 'foo'];
-        })->chunk(200)->each(function ($chunk) {
-            PrunableTestModel::insert($chunk->all());
-        });
+        })->all());
 
-        $count = (new PrunableTestModel)->pruneAll();
+        $count = (new PrunableTestModel)->pruneAll(10);
```

Comment:
Divide everything by 100 here since you don't need to test 5000 models in order to integration-test pruning.

This specific `pruneAll()` function receives a `chunkSize` param which I set manually now since the testcase has become much smaller (for each chunk a prune is executed so I need to change this to `1000(default)/100=10` otherwise the `ModelsPruned::class` event is not dispatched enough times).